lib/matplotlib/axes/_axes.py

```diff
@@ -4925,7 +4925,7 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
             A `.PolyCollection` defining the hexagonal bins.
 
             - `.PolyCollection.get_offsets` contains a Mx2 array containing
-              the x, y positions of the M hexagon centers.
+              the x, y positions of the M hexagon centers in Axes coordinates.
```

Comment:
```suggestion
              the x, y positions of the M hexagon centers in Data coordinates.
```

I don't think `Axes` is correct here.

Reply:
Thanks for catching - fixed.